bin/pacbio_barcode_check.py

```diff
@@ -40,44 +47,48 @@ def check_if_barcodes_exist_in_barcodes_fasta(barcodes_list, barcodes_fasta_path
     barcode_names_in_fasta = [n.split(">")[1] for n in barcodes_fasta_data if n.startswith(">")]
     for barcode in barcodes_list:
         if barcode not in barcode_names_in_fasta:
-            # sys.stderr.write(f"The PacBio multiplexing barcode ({barcode}) was not found in the barcode sequences file ({barcodes_fasta_path})\n")
-            print("NO BARCODES, KILL PIPELINE")
+            sys.stderr.write(
+                f"The PacBio multiplexing barcode ({barcode}) was not found in the barcode sequences file ({barcodes_fasta_path})\n"
+            )
             sys.exit(1)
 
+    # If this print statement is reached, all user-supplied codes are present.
+    print("BARCODES FOUND\n")
```

Comment:
This should say something like "The query barcodes exist in the barcodes database file" to make it clear that "barcodes found" here doesn't mean that barcode contamination was found in the assembly

Reply:
Done